### PR TITLE
NAICS industry classification

### PR DESCRIPTION
**Pull request by karsov**
# Description

## What

Add support for NAICS as a valid identifier system.

## Why

JIRA ticket UPPSF-1942

## Scope and particulars of this PR (Please tick all that apply)

- [ ] Tech hygiene (dependency updating & other tech debt)
- [ ] Bug fix
- [x] Feature
- [ ] Documentation
- [ ] Breaking change
- [ ] Minor change (e.g. fixing a typo, adding config)

___
This Pull Request follows the rules described in our [Pull Requests Guide](https://github.com/Financial-Times/upp-docs/tree/master/guides/pr-guide)

### CODE REVIEW

File: helm/public-concordances-api/templates/service.yaml [hunk 1]
```diff
@@ -3,11 +3,11 @@
 kind: Service
 apiVersion: v1
 metadata:
-  name: {{.Values.service.name}} 
+  name: {{.Values.service.name}}-orig
```
Comment (epavlova) [inline helm/public-concordances-api/templates/service.yaml:6]: Is the idea to go with that configuration to production? Saying "that configuration" I mean the last commit called `Shadowise`? Or the idea is to test it on stage environment?

File: concordances/cypher_test.go [hunk 2]
```diff
@@ -262,272 +262,224 @@ var unconcordedBrandTMEUPP = Concordance{
...
+				Authority:       "http://api.ft.com/system/NAICS",
+				IdentifierValue: "5111"},
+		},
+	},
 }
 
-func TestNeoReadByAuthority_ManagedLocation(t *testing.T) {
-	assert := assert.New(t)
-	db := getDatabaseConnection(t, assert)
+func TestNeoReadByConceptID(t *testing.T) {
```
Comment (epavlova) [inline concordances/cypher_test.go:307]: So much clearer with the table tests. Very nice!

File: concordances/cypher.go [hunk 3]
```diff
@@ -136,6 +143,19 @@ func (pcw CypherDriver) ReadByAuthority(authority string, identifierValues []str
 			},
 			Result: &results,
 		}
+	} else if authorityProperty == "NAICS" {
```
Comment (epavlova) [inline concordances/cypher.go:146]: Totally optional - the bigger `else if` is screaming `switch` statement.

File: helm/public-concordances-api/templates/service.yaml (hunk 1 above)
Comment (karsov) [reply helm/public-concordances-api/templates/service.yaml:6]: I was thinking only Staging. There is no problem to go to prod, but the change is not very big.

File: README.md [hunk 4]
```diff
@@ -1,48 +1,46 @@
...
-    - GET /__build-info
-    - GET /__gtg 
+- GET `/__health`
+- GET `/__build-info`
+- GET `/__gtg`
 
 ## Error handling
-[Run book](https://biz-ops.in.ft.com/System/public-concordances-api) - [Panic guide](https://sites.google.com/a/ft.com/universal-publishing/ops-guides/panic-guides/concordances-read)
+
+[Runbook](https://biz-ops.in.ft.com/System/public-concordances-api) - [Panic guide](https://sites.google.com/a/ft.com/universal-publishing/ops-guides/panic-guides/concordances-read)
```
Comment (ivan-p-nikolov) [inline README.md:41]: **question:** ‏Should we point to the runbook in stead of bizops?
```suggestion
[Runbook](https://runbooks.ftops.tech/public-concordances-api) - [Panic guide](https://sites.google.com/a/ft.com/universal-publishing/ops-guides/panic-guides/concordances-read)
```

**Review by ivan-p-nikolov — APPROVED** [review]

File: README.md (hunk 4 above)
Comment (karsov) [reply README.md:41]: Sure. I was fixing only linter errors so I didn't notice this but I'll fix it now.

Comment (karsov) [reply README.md]: Updated.

**Comment by karsov** [comment]
I've removed the "Shadowise" commit, everything looks fine on Staging.

**Review by epavlova — APPROVED** [review]

**Review by ivan-p-nikolov — APPROVED** [review]

**Pull request merged**